release.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -eq 0 ]
+then
+	echo "No version supplied. Run \`make release VERSION=X.X\`"
+	exit 1
+fi
+
+version=$1
+
+echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$" && echo "match" || echo "no match"
+
+if ! echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$"
+then
+	echo "Invalid version: ${version}"
+	echo "Please specify a semantic version prefixed with a v (e.g. v0.0.0)."
+	exit 1
+fi
+
+platforms="linux windows darwin"
+build_dir=./bin
+name=terraform-validator
+ldflags="-X github.com/GoogleCloudPlatform/terraform-validator/tfgcv.buildVersion=${version}"
+release_bucket=terraform-validator
+
+# Build release versions
+for platform in ${platforms}; do
+	echo "Building version ${version} for ${platform}..."
+	GO111MODULE=on GOOS=${platform} GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "${ldflags}" -o "${build_dir}/${name}-${platform}-amd64" .
```

Comment:
should we add support for apple silicon or other architectures?

Reply:
Possibly? I don't know that we have a clear signal that we're missing particular platforms... But either way, I think that's outside the scope of this PR.